Common/UnitDefinitions/FuelEfficiency.json

```diff
@@ -1,55 +1,84 @@
+  "XmlDocRemarks": "https://en.wikipedia.org/wiki/Fuel_efficiency",
+  "BaseDimensions": {
+    "L": -2
+  },
+  "Units": [
+    {
+      "SingularName": "LiterPer100Kilometers",
+      "PluralName": "LitersPer100Kilometers",
+      "FromUnitToBaseFunc": "100 / {x}",
+      "FromBaseToUnitFunc": "100 / {x}",
+      "Localization": [
+        {
           "Culture": "en-US",
           "Abbreviations": [
             "l/100km"
           ]
-        }]
-      },
-      {
-        "SingularName": "MilePerUsGallon",
-        "PluralName": "MilesPerUsGallon",
-        "FromUnitToBaseFunc": "(100 * 3.785411784) / (1.609344 * {x})",
-        "FromBaseToUnitFunc": "(100 * 3.785411784) / (1.609344 * {x})",
-        "Localization": [{
+        }
+      ]
+    },
+    {
+      "SingularName": "MilePerUsGallon",
+      "PluralName": "MilesPerUsGallon",
+      "FromUnitToBaseFunc": "{x} * 1.609344 / 3.785411784",
+      "FromBaseToUnitFunc": "{x} * 3.785411784 / 1.609344",
+      "Localization": [
+        {
           "Culture": "en-US",
           "Abbreviations": [
             "mpg (U.S.)"
           ]
-        }]
-      },
-      {
-        "SingularName": "MilePerUkGallon",
-        "PluralName": "MilesPerUkGallon",
-        "FromUnitToBaseFunc": "(100 * 4.54609188) / (1.609344 * {x})",
-        "FromBaseToUnitFunc": "(100 * 4.54609188) / (1.609344 * {x})",
-        "Localization": [{
+        }
+      ]
+    },
+    {
+      "SingularName": "MilePerUkGallon",
+      "PluralName": "MilesPerUkGallon",
+      "FromUnitToBaseFunc": "{x} * 1.609344 / 4.54609",
+      "FromBaseToUnitFunc": "{x} * 4.54609 / 1.609344",
+      "Localization": [
+        {
           "Culture": "en-US",
           "Abbreviations": [
             "mpg (imp.)"
           ]
-        }]
-      },
-      {
-        "SingularName": "KilometerPerLiter",
-        "PluralName": "KilometersPerLiter",
-        "FromUnitToBaseFunc": "100 / {x}",
-        "FromBaseToUnitFunc": "100 / {x}",
-        "Localization": [{
+        }
+      ]
+    },
+    {
+      "SingularName": "KilometerPerLiter",
+      "PluralName": "KilometersPerLiter",
+      "FromUnitToBaseFunc": "{x}",
+      "FromBaseToUnitFunc": "{x}",
+      "Localization": [
+        {
           "Culture": "en-US",
           "Abbreviations": [
             "km/l"
           ]
-        }]
-      }
-    ]
-  }
\ No newline at end of file
+        }
+      ]
+    },
+    {
+      "SingularName": "MeterPerCubicMeter",
+      "PluralName": "MetersPerCubicMeter",
+      "BaseUnits": {
+        "L": "Meter"
+      },
+      "FromUnitToBaseFunc": "{x} / 1e6",
+      "FromBaseToUnitFunc": "{x} * 1e6",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [
+            "m/m³"
+          ]
+        }
+      ]
+    }
```

Comment:
I only added this because of the `UnitSystem.SI` coverage, but I'm not really _set_ on having it: you decide.